cmd/list.go

```diff
@@ -222,7 +222,7 @@ func (cooked cookedListCmdArgs) HandleListContainerCommand() (err error) {
 		}
 	}
 
-	traverser, err := InitResourceTraverser(source, cooked.location, &ctx, &credentialInfo, nil, nil,
+	traverser, err := InitResourceTraverser(source, cooked.location, &ctx, &credentialInfo, common.ESymlinkHandlingType.Skip(), nil,
```

Comment:
During traverser we need the actual type as to follow or preserve, why do we pass skip() here?

Reply:
We need to know how we're handling symlinks. If we're passing in `Follow`, the local traverser has some specific behaviour to follow symlinks-- We could avoid this if we handled following in the STE in the future, but given existing functionality it may be best to delay that into 10.19/20.

For skip-- We could present symlinks as a part of a listing call, but we don't include folder stubs anyway, so, I figured same-same on that general "special blobs" concept.